browser/src/control/jsdialog/Widget.ColorPicker.ts

```diff
@@ -103,9 +124,20 @@ function createColor(
 	color.setAttribute('name', colorItem);
 	color.setAttribute('index', index);
 	color.tabIndex = 0;
+	color.innerHTML = isCurrent ? '&#149;' : '&#160;';
 	if (themeData) color.setAttribute('theme', themeData);
 
-	color.innerHTML = isCurrent ? '&#149;' : '&#160;';
+	// Set color tooltips
+	if (themeData) {
+		const themeColors: ThemeColor[][] =
+			window.app.colorPalettes['ThemeColors'].colors;
+		const flattened: ThemeColor[] = themeColors.flat();
```

Comment:
can we do flattened version once and reuse it ? createColor is for single element, we do it in the loop